src/plugins/curator/components/promotion-details-fields/Slots.jsx

```diff
@@ -0,0 +1,216 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Sortable from 'sortablejs';
+import { Field, FieldArray, FormSection } from 'redux-form';
+import TextField from '@triniti/cms/components/text-field';
+import UncontrolledTooltip from '@triniti/cms/plugins/common/components/uncontrolled-tooltip';
+import humanizeEnums from '@triniti/cms/utils/humanizeEnums';
+import SelectField from '@triniti/cms/components/select-field';
+import { Button, Col, FormGroup, Icon, ListGroupItem } from '@triniti/admin-ui-plugin/components';
+import WidgetPickerField from '@triniti/cms/plugins/curator/components/widget-picker-field';
+import SlotRendering from '@triniti/schemas/triniti/curator/enums/SlotRendering';
+
+const renderingOptions = humanizeEnums(SlotRendering, { format: 'map', except: [SlotRendering.UNKNOWN] });
+
+export default class Slots extends React.Component {
+  static propTypes = {
+    borderClass: PropTypes.string,
+    fields: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
+    readOnly: PropTypes.bool,
+  };
+
+  static defaultProps = {
+    borderClass: null,
+    readOnly: false,
+  };
+
+  constructor(props) {
+    super(props);
+    this.handleAdd = this.handleAdd.bind(this);
+    this.handleMoveDown = this.handleMoveDown.bind(this);
+    this.handleMoveUp = this.handleMoveUp.bind(this);
+  }
+
+  componentDidMount() {
+    this.sortable = Sortable.create(this.sortableListRef, {
+      animation: 150,
+      chosenClass: 'active',
+      draggable: '.sortable-slots',
+      filter: '.js-remove',
+      handle: '.sort-handle',
+      onUpdate: () => {
+        const { fields } = this.props;
+        const newSlotOrder = this.sortable.toArray();
+        fields.removeAll();
+        newSlotOrder.forEach((slot) => {
+          try {
+            fields.push(JSON.parse(slot));
+          } catch (e) {
+            fields.push();
+          }
+        });
+      },
+    });
+  }
+
+  handleAdd() {
+    const { fields } = this.props;
+    fields.push({ name: "" });
```

Comment:
@albertgcsula - nit: use single quotes for strings

Reply:
fixed